model/converter/thrift/jaeger/sampling_from_domain_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/jaegertracing/jaeger-idl/proto-gen/api_v2"
-	"github.com/jaegertracing/jaeger/thrift-gen/sampling"
+	"github.com/jaegertracing/jaeger-idl/thrift-gen/sampling"
```

Comment:
please don't make this change in the same PR. It make it impossible to validate if the new aliases are working. We need to merge a change where only aliases are introduced, and then next PR to swap the imports.